allauth/socialaccount/providers/metamask/templates/metamask/auth.html

```diff
@@ -0,0 +1,73 @@
+{% load static %}
+<div id="personalSign">  </div>
+<HR>
+<div id="personalSignResult">  </div>
+<HR>
+<script type="text/javascript">
+if (typeof jQuery == 'undefined') {
+  document.write('<script type="text/javascript" src="https://code.jquery.com/jquery-3.6.0.min.js"\n' +
+    '  integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4="\n' +
+    '  crossorigin="anonymous"><\/script>');
+  }
+</script>
+<script src="https://cdn.jsdelivr.net/npm/web3@latest/dist/web3.min.js"></script>
+<script type="text/javascript">
+async function loginwithtoken(token,account,accounts,next,process){
+    var web3 = new Web3(Web3.givenProvider);
+    const personalSignResult = document.getElementById('target');
+    try {
+      const from = account;
+      let message = web3.utils.utf8ToHex(token);
+      const sign = await ethereum.request({
+        method: 'personal_sign',
+        params: [message, from,''],
+      });
+      var res = sign
+    } catch (err) {
+      console.error(err);
+      personalSign.innerHTML = `Error: ${err.message}`;
```

Comment:
This variable (`personalSign`)  does not seem to be defined?

Reply:
Dropping this.